submissions/usides/friends-app/js/main.js

```diff
@@ -0,0 +1,159 @@
+const HUMANS_LINK =
+  'https://randomuser.me/api/?nat=us&results=36&inc=gender,name,picture,dob&seed=06b64d62bc01d624a';
+const BAD_STATUS = 200;
+
+const searchBtn = document.getElementById('search-btn');
+const rightPane = document.getElementById('right-pane');
+const cardsHolder = document.getElementById('cards-holder');
+const optionGroup = document.getElementById('option-group');
+const filterButtons = document.querySelectorAll('button[data-filter]');
+const nameInput = document.getElementById('name-input');
+const errorNotice = document.getElementById('error-notice');
+const loader = document.getElementById('loader');
+
+let humansOriginState;
+
+const currentState = {
+  humans: undefined,
+  sort: 'a-z',
+  filter: 'none',
+  searchValue: '',
+};
+
+const handleSearchButton = function () {
+  rightPane.classList.toggle('visible');
+};
+
+let timer;
+const handleInput = function () {
+  if (timer) clearTimeout(timer);
+  timer = setTimeout(() => {
+    currentState.searchValue = this.value.toLowerCase();
+    filterHumans(currentState.filter);
+    sortHumans(currentState.sort);
+    updateCards(currentState.humans);
+    timer = null;
+  }, 500);
```

Comment:
I mean you can create a variable for `500` value

Reply:
Done.